overlay.d/05rhcos/usr/libexec/coreos-cryptfs

```diff
@@ -136,7 +136,9 @@ try_open() {
 
         # TODO: make this idempotent, and don't error out if
         # we can't resize.
-        growpart "${parent_device}" "${partition}" || true
+        # Investigation OCPBUGS-33124 proved the growpart causes the partition disappears.
+        # Hence removed from coreos-cryptfs.
+        # growpart "${parent_device}" "${partition}" || true
```

Comment:
No need to comment it out. Let's just drop that line and that's it.

Reply:
Added the commit message and dropped the growpart.